lib/gc/Transforms/Pipeline.cpp

```diff
@@ -142,6 +142,7 @@ void populateLoweringToLLVMPasses(mlir::OpPassManager &pm) {
   pm.addNestedPass<func::FuncOp>(createConvertMathToLLVMPass());
   pm.addPass(createConvertMathToLibmPass());
   pm.addNestedPass<func::FuncOp>(createArithToLLVMConversionPass());
+  pm.addPass(createConvertVectorToLLVMPass());
```

Comment:
Also need to add `createLowerTileVectorPass()` in within tensor related passes?

Reply:
Yes. But I don't want to add it directly to our pipeline at the moment. Because it will unroll all instructions by default, generate a large IR, and bring compilation overhead.   Like this [issue](https://github.com/intel/graph-compiler/issues/251#issuecomment-2290856269) mentioned.

Anyone who wants to test lower to vector can add by ownself？

Reply:
I see, thanks.